docs/source/top_level_elements/location.rst

```diff
@@ -0,0 +1,203 @@
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2017033007> .
+
+physicalLocation as string (special collections)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/roth:4245/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennessee, Knoxville. Special Collections</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2014027633> .
+
+physicalLocation as string (libraries)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**The University of Tennessee Libraries, Knoxville** 574 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/fbpro:94819/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>The University of Tennessee Libraries, Knoxville</physicalLocation>
+    </location>
+
+**University of Tennessee Knoxville. Libraries** 664 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/tatum:609/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennessee Knoxville. Libraries</physicalLocation>
+    </location>
+
+**University of Tennesse Knoxville. Libraries** 397 instances
+
+https://digital.lib.utk.edu/collections/islandora/object/tdh:8781/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation>University of Tennesse Knoxville. Libraries</physicalLocation>
+    </location>
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/n80003889> .
+
+physicalLocation and shelfLocator
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/scopes:1258/datastream/MODS/view
+
+.. code-block:: xml
+
+    <location>
+        <physicalLocation valueURI="http://id.loc.gov/authorities/names/no2014027633">University of Tennessee, Knoxville. Special Collections</physicalLocation>
+        <shelfLocator>Box 5, Folder 8</shelfLocator>
+    </location>
+
+In most cases, especially those under our purview, we will likely opt to drop box and folder number references.
+
+.. code-block:: turtle
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators> .
+    @prefix opaque: <http://opaquenamespace.org/ns/> .
+
+    <https://example.org/objects/1> relators:rps <http://id.loc.gov/authorities/names/no2014027633> .
+
+physicalLocation with holdingSimple and holdingExternal
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+https://digital.lib.utk.edu/collections/islandora/object/volvoices:2199/datastream/MODS/view
```

Comment:
Particularly with the Volunteer Voices Collection, we are going to need to think about how physicalLocation works with recordContentSource. I had forgotten that this collection only lists the contributing institution in physicalLocation and not in recordContentSource (which differs from collections like cdf and egypt). The issue is making sure that the institution (Un. of Memphis in this case) gets credit in DPLA for sharing this item instead of us. As I have laid out the mapping in recordContentSource, we would get credit for this item.